styles/books/neuroscience/book.scss

```diff
@@ -78,7 +78,13 @@ $ChapterIntroType: side;
 
 
 //Lists
+@include use('PageLists', 'carnival:::PageListsShape');
+@include use('CompositePageLists', 'carnival:::CompositePageListsShape');
+@include use('TableLists', 'carnival:::TableListShape');
+@include use("NotesLists", "carnival:::NotesListsShape");
 
+//EOC
+@include use('KeyTerms', 'carnival:::KeyTermsShape');
```

Comment:
Terms from the issue are terms that appears within text, not in the EOC section. They are styled in `common-styles`. But I think we can leave this here. Maybe we can add just a note in issue for EOC sections that Key Terms is styled in this PR.